cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -842,6 +851,34 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
   toObject _verb _ =
     mkObject [ "kind" .= String "TraceLocalTxSubmissionServerEvent" ]
 
+instance (Show (GenTx blk), Show (GenTxId blk), Show (ApplyTxErr blk))
+      => ToObject (TraceEventMempool blk) where
+  toObject _verb (TraceMempoolAddTxs txs mpSz) =
+    mkObject
+        [ "kind" .= String "TraceMempoolAddTxs"
+        , "txsAdded" .= String (pack $ show $ txs)
+        , "mempoolSize" .= String (pack $ show $ mpSz)
```

Comment:
So using `show` here kind of sucks. Preferably, we'd want objects like `GenTx`s, `ApplyTxErr`s, and `MempoolSize`s to all have `ToJSON`/`ToObject` instances for structured logging purposes.

@dcoutts @Jimbo4350: Should the instances for the aforementioned data types be defined in `ouroboros-consensus` or here in `cardano-node`? Immediately, I'm thinking to define orphans here in this module since logging output is more relevant to `cardano-node`.